plugins/lsp/language-servers/default.nix

```diff
@@ -620,6 +620,12 @@ let
       description = "tinymist for Typst";
       settingsOptions = import ./tinymist-settings.nix { inherit lib helpers; };
     }
+    {
+      name = "ts-ls";
+      serverName = "ts_ls";
+      description = "tsserver for TypeScript";
+      package = "typescript-language-server";
+    }
     {
       name = "tsserver";
       description = "tsserver for TypeScript";
```

Comment:
Should nixvim print a deprecation warning when this used?

Reply:
Alternatively, seeing as both use the same package under the hood perhaps we could simply make this a `mkRenameOptioModule` alias pointing to the new name?